controller/main.py

```diff
@@ -77,8 +77,9 @@ async def serve(self):
 
 
 class PatternGenerator:
-    def __init__(self, patter_selector):
+    def __init__(self, patter_selector, color_selector=None):
         self.patter_selector = patter_selector
+        self.color_selector = color_selector
```

Comment:
Are you planning to add this in the future?

Reply:
Doh! Old code from an alternate implementation I tried. Removing